rs/config/src/subnet_config.rs

```diff
@@ -451,13 +444,15 @@ impl CyclesAccountManagerConfig {
             http_request_quadratic_baseline_fee: Cycles::new(60_000),
             http_request_per_byte_fee: Cycles::new(400),
             http_response_per_byte_fee: Cycles::new(800),
-            // This effectively disables the storage reservation mechanism on
-            // verified application subnets.
-            max_storage_reservation_period: Duration::from_secs(0),
+            max_storage_reservation_period: Duration::from_secs(300_000_000),
             default_reserved_balance_limit: DEFAULT_RESERVED_BALANCE_LIMIT,
         }
     }
 
+    pub fn verified_application_subnet() -> Self {
+        Self::application_subnet()
+    }
+
```

Comment:
For consistency with the subnet config I'm moving the configuration options to `Self::application_subnet` and use it inside `Self::verified_application_subnet`.